bindings/ruby/tests/steps/binding.rb

```diff
@@ -36,7 +38,7 @@
 end
 
 Then("The blocking file {string} must have content {string}") do |string, string2|
-  pending # Write code here that turns the phrase above into concrete actions
+  @op.read(string).map { |num| num.chr }.join == string2
```

Comment:
Does Ruby have a native bytes type? The method we demonstrated here does not seem to be the typical approach used by Ruby users.

Take aws ruby s3 sdk as an example:

https://docs.aws.amazon.com/sdk-for-ruby/v1/api/AWS/S3/S3Object.html

```ruby
obj.write('Hello World!')
obj.read
#=> 'Hello World!'
```

Reply:
Please always remember our VISION: `Access data freely, painlessly, and efficiently.`

Reply:
> Does Ruby have a native bytes type? 

The std library seems favor just use String on representing bytes like:

https://ruby-doc.org/core-2.5.0/IO.html#method-c-read

The string in ruby can contains arbitary characters inside, do not require it must be utf-8 encoded.

> A String object has an arbitrary sequence of bytes, typically representing text or binary data. A String object may be created using [String::new](https://ruby-doc.org/3.2.1/String.html#method-c-new) or as literals.

Reply:
Now, the `Vec<u8>` in `read` and `write` has been replaced with `String`.